db/model/subject.js

```diff
@@ -440,9 +446,7 @@ module.exports = function subject(seq, dataTypes) {
 
               return resolve(inst);
             })
-            .catch((err) => {
-              return reject(err);
-            });
+            .catch(reject);
```

Comment:
ok. somehow coveralls check is stalled. Have you seen this problem before @shriramshankar ?